### PR TITLE
dev-util/jenkins-bin:  version bump 2.11

### PR DESCRIPTION
**Pull request by bdouxx**
Package-Manager: portage-2.2.28

many evolution since jenkins-bin-2.5

### CODE REVIEW

File: dev-util/jenkins-bin/jenkins-bin-2.11.ebuild [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```
Comment (SoapGentoo) [inline dev-util/jenkins-bin/jenkins-bin-2.11.ebuild]: why no `EAPI=6`?

**Comment by mgorny** [comment]
@mrueg, @gentoochainsaw

File: dev-util/jenkins-bin/jenkins-bin-2.11.ebuild (hunk 1 above)
Comment (bdouxx) [inline dev-util/jenkins-bin/jenkins-bin-2.11.ebuild]: hi
It's my first pull-request, I wasn't sure about the process.
So I preferred not to make modification on the ebuild.

Comment (bdouxx) [inline dev-util/jenkins-bin/jenkins-bin-2.11.ebuild]: hi
It's my first pull-request, I wasn't sure about the process.
So I preferred not to make modification on the ebuild.

**Pull request closed without merging**